tests/operator_test.go

```diff
@@ -887,15 +888,18 @@ var _ = Describe("ALL Operator tests", func() {
 					serverSecrets := getSecrets(serverSecretNames)
 
 					for _, s := range append(caSecrets, serverSecrets...) {
+						fmt.Fprintf(GinkgoWriter, "Loop 1\n")
```

Comment:
Might want to take these debug statements out?

Reply:
I actually intended to keep them, very hard to debug otherwise

Reply:
Okay, could you make them a little more descriptive for our future selves then, loop 1,2,3 doesn't mean much without context. I certainly don't understand what they mean.

Reply:
Done